Wolox/WLXQuickBloxUtils-sample
# login view

@guidomb 
## Sumary

**Trello card**: https://trello.com/c/mIIKQZHB/9-pantalla-de-login

Login view
## Test Plan
- Open the app
- Login with username 'asd@asd.com' password 'asdasdasd', verify that no error message is shown
- Sign up with any unexisting user, verify that no error message appears
- Login with an unexistant user, verify that an error message appears
- Login with an invalid email or password, verify that an error message appears 
## Screenshots

![ios simulator screen shot jul 10 2015 5 57 31 pm](https://cloud.githubusercontent.com/assets/5349650/8628929/3bcefaba-272d-11e5-9e77-f8a234fa3f3d.png)
## Known issues

View model tests pending
